docs/adv-rumors.rst

```diff
@@ -2,21 +2,17 @@ adv-rumors
 ==========
 
 .. dfhack-tool::
-    :summary: Improves the rumors menu in adventure mode.
-    :tags: unavailable
+    :summary: Improves the conversation menus in Adventure mode.
+    :tags: adventure interface
 
-In adventure mode, start a conversation with someone and then run this tool
-to improve the "Bring up specific incident or rumor" menu. Specifically, this
-tool will:
+In adventure mode, start a conversation with someone. This tool will:
 
-- Move entries into a single line to improve readability
-- Add a "slew" keyword for filtering, making it easy to find your kills and not
-  your companions'
-- Trim repetitive words from the text
+- Add all words to keywords for easier filtering/searching everywhere
+- Additional 'slay' keyword for all relevant instances
 
-Usage
------
+Overlay
+-------
 
-::
-
-    adv-rumors
+This script also provides an overlay that is managed by the `gui/overlay` framework.
+When the `gui/adv-rumors` overlay is enabled, the script will automatically run on the convesration screen,
```

Comment:
the name of the overlay will be `adv-rumors` then a `.`, then the name you assigned in the `OVERLAY_WIDGETS` table (see https://github.com/DFHack/scripts/pull/442/files#r1622685309)